site/docs/components/card/accessibility.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+#### Default Cards focus sequence
+
+For default Cards group, focus moves through interactive elements within the cards in a logical tabbing order.
+
+#### Interactable cards focus sequence
+
+For Interactable cards group, focus moves through cards in a logical tabbing order.
```

Comment:
is there any guidance around interactable cards containing interactable elements?

Reply:
definitely need to add this point, it was a big one when Ivan and I worked on it

I think we can also add in 'when not to use' when cards have multiple interactable elements rather than the whole thing intended to be interacted with. @ivan-calderon will know best here

Reply:
Thank you @alycrys! We just added the 'Interactable elements' to guide on the above.